docs/smart_rollups.md

```diff
@@ -27,10 +27,11 @@ await op.confirmation();
 
 ##### please note
 - `pvmKind` at this time the only PVM supported is `wasm_2_0_0`
-- `kernel` is passed as a hexadecimal string examples can be found at [this tezos docs endpoint](https://tezos.gitlab.io/mumbai/smart_rollups.html?highlight=smart%20rollup%20originate#id1)
+- `kernel` is passed as a hexadecimal string examples can be found at [this tezos docs endpoint](https://tezos.gitlab.io/oxford/smart_rollups.html?highlight=smart%20rollup%20originate#id1)
 - `parametersType` is a MichelsonV1Expression to define the type.
+-  **Please note** `originationProof` is needed origination in `batch` which can be obtained with the `rpc.getOriginationProof({kernel, kind: PVMKind})`
```

Comment:
I thought this param was removed in Oxford?

Reply:
Thanks for caching, will remove. It's removed in [oxford branch]( https://github.com/ecadlabs/taquito/pull/2661/files#diff-d8805c22ac4e0fde66a9c098bdd50de333641a472ddae1d37b0a74f042fb2b67L32) seems overwriting by auto-merging. 
<img width="1467" alt="image" src="https://github.com/ecadlabs/taquito/assets/106410553/694e6dd1-d18f-4836-8c22-47c10fde9ca1">